### ISSUE TITLE
Foreign accent syndrome

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11606]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 30606457 Foreign Accent Syndrome (FAS) is a relatively rare motor speech disorder characterized by speech errors which are perceived as a foreign accent by members of the same language community as the patient.

PMID: 34274725 Foreign accent syndrome (FAS) is a rare acquired speech disorder wherein an individual’s spoken
accent is perceived as “foreign.” Most reported cases involve left frontal brain lesions, but it is known that various
other lesions can also cause FAS. 

**Parent term (use hpo.jax.org/app)**
Speech articulation difficulties 
HP:0009088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:66624 PANDAS 

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
